### PR TITLE
Feature/reskin calendar

### PR DESCRIPTION
**Pull request by asgalecki**

### CODE REVIEW

**Comment by asgalecki** [comment]
calendarReskin

File: src/components/ExpandableCalendar.tsx [hunk 1]
```diff
@@ -65,6 +66,12 @@ export const ExpandableCalendar = (props: ExpandableCalendarProps & RNCalendarPr
     getInitialCalendarHeight(isScreenHeightShort, hasUserSeenCalendar || false)
   )
 
+  // Calendar footer arrow animation
```
Comment (iceu-bb) [inline src/components/ExpandableCalendar.tsx]: redundant comment

File: src/components/ExpandableCalendar.tsx [hunk 2]
```diff
@@ -225,6 +248,17 @@ export const ExpandableCalendar = (props: ExpandableCalendarProps & RNCalendarPr
               />
             </Box>
           </Animated.View>
+          <BaseOpacity
+            hitSlop={{ top: 30, right: 30, bottom: 30, left: 30 }}
+            onLayout={(e) => trackCalendarHeight(e)}
+            onPress={() => toggleOnPress()}
```
Comment (iceu-bb) [inline src/components/ExpandableCalendar.tsx]:
```suggestion
            onLayout={trackCalendarHeight}
            onPress={toggleOnPress}
```

File: src/screens/calendar/components/GoUpDownButton.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      width={40}
+      height={40}
+      position="absolute"
+      bottom={84}
+      right={24}
+      backgroundColor="special"
+      borderRadius="full"
+      justifyContent="center"
+      alignItems="center"
+      onPress={() => props.onPress()}>
```
Comment (iceu-bb) [inline src/screens/calendar/components/GoUpDownButton.tsx]:
```suggestion
      onPress={props.onPress}>
```

**Review by iceu-bb — APPROVED** [review]

**Pull request merged**